Assets/HoloToolkit/Utilities/Scripts/StabilizationPlaneModifier.cs

```diff
@@ -21,8 +21,23 @@ public class StabilizationPlaneModifier : Singleton<StabilizationPlaneModifier>
         [Tooltip("Used to temporarily override the location of the stabilization plane.")]
         public Transform TargetOverride;
 
-        [Tooltip("Keeps track of position-based velocity for the target object.")]
-        public bool TrackVelocity = false;
+        [SerializeField, Tooltip("Keeps track of position-based velocity for the target object.")]
+        private bool trackVelocity = false;
+        public bool TrackVelocity
+        {
+            get
+            {
+                return trackVelocity;
+            }
+            set
+            {
+                trackVelocity = value;
+                if (TargetOverride)
```

Comment:
This is looking better, but now I realize you've got the same issue with late sets of the `TargetOverride`. If that variable changes we should clear the previous position as well.